### PR TITLE
merge retest into re

### PR DESCRIPTION
**Pull request by alfredh**

### CODE REVIEW

File: test/test.c [hunk 1]
```diff
@@ -0,0 +1,1104 @@
...
+static int dup_stderr;
+
+enum test_mode test_mode = TEST_NONE;
+
+
+static void hide_output(void)
+{
+	dup_stdout = dup(fileno(stdout));
+	dup_stderr = dup(fileno(stderr));
+	int fd_out = open("stdout.out", O_WRONLY | O_CREAT);
```
Comment (sreimers) [inline test/test.c:282]:
```suggestion
	int fd_out = open("stdout.out", O_WRONLY | O_CREAT, S_IWUSR | S_IRUSR);
```

File: test/test.c [hunk 2]
```diff
@@ -0,0 +1,1104 @@
...
+	int fd_err = open("stderr.out", O_WRONLY | O_CREAT);
```
Comment (sreimers) [inline test/test.c:283]:
```suggestion
	int fd_err = open("stderr.out", O_WRONLY | O_CREAT, S_IWUSR | S_IRUSR);
```

**Comment by alfredh** [comment]
thanks for the comments, code was updated.

Please take a look at the cmake changes.

**Comment by alfredh** [comment]
https://github.com/baresip/baresip-win32/pull/11

**Pull request merged**